### ISSUE TITLE
Add validation rules to account creation

### ISSUE THREAD

**Issue by aaroncox** [aaroncox/vessel#65]
Currently there are no rules while creating an account, and when creating an account with an invalid name, it looks like it creates them (but nothing happens on-chain).

**Comment by eastmaels** [comment]
Yes, this is confirmed. I tried creating one today and it didn't give me an error when the length of the account name exceeds a certain number of characters.

The account name I tried to create was `eastmael.autovote`. When I tried to create it via steemconnect, I was only allowed to type `eastmael.autovot` (no **e** at the end).
